src/index.js

```diff
@@ -40,16 +38,19 @@ if (process.env.NODE_ENV === "production") {
 // a few cycles before trying again.
 let checkCounter = 0;
 const checkConfigExists = () => {
-  if (!window.jujuDashboardConfig && checkCounter < 5) {
-    checkCounter++;
-    setTimeout(checkConfigExists, 500);
-    return;
+  if (!window.jujuDashboardConfig) {
+    if (checkCounter < 5) {
```

Comment:
This is a driveby fix to fix #589

Reply:
Can we cherrypick this out into a separate branch and get it landed?